src/siembol/templates/configmap-ui.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
```

Comment:
not needed if we use config from docker image

Reply:
Addressing in a later issue